core/build.sbt

```diff
@@ -1,7 +1,9 @@
-import Dependencies._
+import Dependencies.{Version, _}
 
 name := "pureconfig-core"
 
+crossScalaVersions += Version.scala30
```

Comment:
It's becoming increasing complex to handle `crossScalaVersions` across projects. Up until recently not all projects supported Scala 2.13; a significant number of them don't support Scala 2.11 anymore and I suppose we'll be introducing Scala 3.0 compilation incrementally over time. Moreover, we don't always use the same syntax, preferring to filter out 2.11 versions using a `~=` rather than `-=`.

At this point, I'm starting to think it may not make sense anymore to have a default for `crossScalaVersions` and we should simply have all projects define it explicitly. What do you think?

Reply:
I agree. It makes it easier to understand which versions a given module is being compiled with.

Reply:
Much better, thanks!